tests/unit/testsuite/Bolt/Boltpay/Block/InfoTest.php

```diff
@@ -58,4 +70,38 @@ public function prepareSpecificInformation()
             ], $data->getData()
         );
     }
+    
+    /**
+     * @test
+     * 
+     * @covers ::displayPaymentMethodTitle
+     */
+    public function displayPaymentMethodTitle_ifOrderPaidWithCreditCard_returnDefaultTitle()
+    {
+        $this->currentMock->expects(self::any())->method('getInfo')->willReturn($this->infoMock);
+        $this->currentMock->expects(self::any())->method('getMethod')->willReturn($this->paymentMock);
+        $this->infoMock->expects(self::once())->method('getAdditionalInformation')->with('bolt_processor')
+            ->willReturn('vantiv');
+        $this->paymentMock->expects(self::once())->method('getTitle')->willReturn('Credit and Debit Card (Powered by Bolt)');
+        $this->assertEquals(
+            'Credit and Debit Card (Powered by Bolt)',
+            $this->currentMock->displayPaymentMethodTitle()
+        );
+    }
+    
+    /**
+     * @test
+     * 
+     * @covers ::displayPaymentMethodTitle
+     */
+    public function displayPaymentMethodTitle_ifOrderPaidWithPaypal_returnPaypalTitle()
+    {
+        $this->currentMock->expects(self::any())->method('getInfo')->willReturn($this->infoMock);
+        $this->infoMock->expects(self::once())->method('getAdditionalInformation')->with('bolt_processor')
+            ->willReturn('paypal');
+        $this->assertEquals(
+            'Bolt-PayPal',
+            $this->currentMock->displayPaymentMethodTitle()
+        );
+    }
```

Comment:
You are missing test for AfterPay and anything that is not PayPal, AfterPay, or Vantiv.

For `afterparty`, we should expect `Bolt-Afterpay`, and for something unexpected like `new-payment-type`, we should be expecting `Bolt-NEW-PAYMENT-TYPE`.

I suggest merging the two presented test to one and using a dataProvider to cover the various cases, (i.e. Vantiv, PayPay, AfterPay, <unexpected>)